### PR TITLE
[DRAFT] feat: gh-action build

### PR DESCRIPTION
**Pull request by krol3**
- Adding unit test using https://github.com/shellspec/shellspec
- Fixing https://github.com/aquasecurity/trivy/discussions/1515"

### CODE REVIEW

**Comment by krol3** [comment]
@simar7 could you take a look ? I used shellspec to write the unit test.

**Comment by simar7** [comment]
> @simar7 could you take a look ? I used shellspec to write the unit test.

Sorry about the delay will take a look shortly.

File: spec/entrypoint_spec.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#shellcheck shell=sh
```
Comment (simar7) [inline spec/entrypoint_spec.sh:1]: This is a good start but for completeness you could add a golden image that has a known footprint of vulnerabilities like this https://hub.docker.com/r/knqyf263/vuln-image

Then you can have more concrete assertions.

**Comment by krol3** [comment]
Closing in benefit of https://github.com/aquasecurity/trivy-action/pull/101

**Pull request closed without merging**